packages/fast-components-react-base/src/tabs/tabs.tsx

```diff
@@ -168,6 +154,26 @@ class Tabs extends Foundation<TabsHandledProps, TabsUnhandledProps, TabsState> {
         return get(this.props, "managedClasses.tabs_tabPanelContent") || "";
     }
 
+    /**
+     * Create tab class names
+     */
+    protected generateTabClassNames(): any {
+        return {
+            tab: `${get(this.props, "managedClasses.tab", "")}`,
```

Comment:
I think instead of "" you want this to be undefined since these are all optional, so something like:

```
const tabClass: string = get(this.props, "managedClasses.tab");

return {
    tab: tabClass,
    // etc.
}
```

Reply:
How's that?